### PR TITLE
bpo-28929: Add to Misc/NEWS

### PR DESCRIPTION
**Pull request by Mariatta**
Mention bpo-28929 in the Documentation section of:

- What's New in Python 3.7.0 alpha 1?
- What's New in Python 3.6.1 release candidate 1?
- What's New in Python 3.5.4 release candidate 1?

http://bugs.python.org/issue28929

### CODE REVIEW

File: Misc/NEWS [hunk 1]
```diff
@@ -700,6 +700,8 @@ C API
 Documentation
 -------------
 
+- bpo-28929: Link the documentation to its source file on GitHub
```
Comment (brettcannon) [inline Misc/NEWS]: Missing a period.

File: Misc/NEWS [hunk 2]
```diff
@@ -909,6 +911,8 @@ Windows
...
```
Comment (brettcannon) [inline Misc/NEWS]: This is unnecessary as the same bit of information is picked up in the 3.7 release notes.

File: Misc/NEWS [hunk 3]
```diff
@@ -4095,6 +4099,17 @@ C API
   instead of TypeError on programmical error in parsing format string.
 
 
+What's New in Python 3.5.4 release candidate 1?
```
Comment (brettcannon) [inline Misc/NEWS]: This is also unnecessary.

**Review by brettcannon — CHANGES_REQUESTED** [review]
We typically don't worry about covering NEWS entries more than once in a single copy of the file since it's redundant information at that point; as long as someone reading the release notes in a release can see the news entry _somewhere_ then that's enough.

**Comment by Mariatta** [comment]
Thanks @brettcannon :) 
I removed the unnecessary entries.

**Comment by Mariatta** [comment]
I suppose it doesn't need backport then. But please re-add the labels if I'm mistaken.

**Comment by brettcannon** [comment]
You can still backport into the appropriate sections, I'm just saying you don't need it more than once in the file at any one time.

And I'm now on vacation so I'll let someone else approve. 😀

**Comment by Mariatta** [comment]
Ah ok I think I got it now 😅 Re-added the backport labels. 

Please enjoy your vacation!

**Review by brettcannon — APPROVED** [review]

**Pull request merged**